Cargo.toml

```diff
@@ -7,9 +7,18 @@ edition = "2021"
 
 [dependencies]
 axum = "0.5.17"
+base64 = "0.13.1"
 chrono = { version = "0.4.19", features = ["serde"] }
+futures = "0.3.25"
+hmac-sha256 = "1.1.5"
+itertools = "0.10.5"
+rand = "0.8.5"
+reqwest = { version = "0.11.12", features = ["json"] }
+rsa = "0.7.2"
+rustypub = { git = "https://github.com/hachyserve/rustypub" }
```

Comment:
do you need a release of this? is it functional enough for this project?

Reply:
ah, I'm not acually making use of it yet: I hit those ownership issues we chatted about in discord and moved back to a minimal set of structs defined here for now. I'll remove it from the Cargo.toml for now :+1: